rpcs3/Emu/CPU/CPUThread.cpp

```diff
@@ -382,17 +382,7 @@ void cpu_thread::operator()()
 		// Check stop status
 		if (!(state & cpu_flag::stop))
 		{
-			try
-			{
-				cpu_task();
-			}
-			catch (const std::exception& e)
-			{
-				sys_log.fatal("%s thrown: %s", typeid(e).name(), e.what());
-				sys_log.notice("\n%s", dump());
-				break;
-			}
-
+			cpu_task();
```

Comment:
cpu_thread::dump() removed.

Reply:
Technically an issue before this pr as well but its because of fmt::throw_exception changes.

Reply:
True, but no idea where to put it. After emergency_exit, you can still look at it I believe.

Reply:
Maybe I'll put it in thread_local finalizer.

Reply:
Well, re-implemented dump(), but something is off, need to debug it a bit.

Reply:
Should work now.